pkg/monitor/monitor.go

```diff
@@ -387,7 +387,15 @@ func (m *Monitor) doesDriverOwnPodVolumes(pod *v1.Pod) (bool, error) {
 		return false, err
 	}
 
-	if len(volumes) == 0 {
+	// DirectAttached Volumes are not considered if pods use those
+	volumeCount := 0
+	for _, volume := range volumes {
+		if !volume.DirectAttached {
+			volumeCount++
+		}
```

Comment:
We might need to add WindowsVolume too in future when we start supporting it. For now this looks ok.